src/app/components/icons/IconCard.tsx

```diff
@@ -39,7 +40,7 @@ export const IconCard: React.FC<IconCardProps> = (props): JSX.Element => {
 
     const { component: Component, name, showLabel, style, selected, iconSize, className } = props;
     return (
-        <div className={`${classes.wrapper} ${selected ? classes.selected : ''} + ${className}`} style={style}>
+        <div className={clsx(classes.wrapper, { [classes.selected]: selected }, className)} style={style}>
```

Comment:
I actually saw a couple instances where we were using the `+` inside of a template literal. You may want to do a quick sweep and see if you can find any more.